sfaira/_settings.py

```diff
@@ -0,0 +1,72 @@
+"""
+Settings class which for example holds paths to cache directories used throughout the code.
+"""
+
+import os
+
+
+class SfairaConfig:
+    """\
+    Config manager for sfaira.
+    """
+
+    def __init__(self):
+        self.sfaira_repo_url = "https://zenodo.org/record/4836517/files/"
```

Comment:
Can we put this into a global namespace constant of this file? That way management of constants in sfaira is a bit easier, ie this could also be factored away from this file in the future.

Reply:
so `self.sfaira_repo_url = SFAIRA_REPO_ULR` and define `SFAIRA_REPO_ULR` outside of the class.